scss/_functions.scss

```diff
@@ -88,14 +88,38 @@
 }
 
 // Color contrast
-@function color-yiq($color, $dark: $yiq-text-dark, $light: $yiq-text-light) {
-  $r: red($color);
-  $g: green($color);
-  $b: blue($color);
+// Based on @ysds implementation
```

Comment:
No reason to mention people here. Would rather like to see some explanation about the color contrast in general here.

Reply:
Or maybe here: https://deploy-preview-30168--twbs-bootstrap.netlify.com/docs/4.3/getting-started/theming/#color-contrast. We could mention something about the 4.5-rule.

Reply:
I improved comments a bit, and will try to document WCAG contrast ratios thing in docs.

Your comment raises another question though: we're not using the YIQ ciolor space anymore… Shouldn't we rename or alias this function?

Reply:
And FWIW, in the "Accessibility" page there's a section to mention insufficient contrast: https://deploy-preview-30168--twbs-bootstrap.netlify.com/docs/4.3/getting-started/accessibility/#color-contrast

I saw many commits improving colors and contrasts in v5: is this still relevant? ping @XhmikosR, I think you're the one who improved contrasts.